deploy/Vagrantfile

```diff
@@ -629,9 +631,7 @@ Vagrant.configure(2) do |config|
     SHELL
 
     server.vm.provision "ansible" do |ansible|
-      ansible_extra_vars = {
-        "jre_lib_path": "/usr/lib/jvm/java-1.8.0-openjdk-1.8.0.131-11.b12.el7.x86_64/jre/lib",
-        "java_home_path": "/usr/lib/jvm/java-1.8.0-openjdk-1.8.0.131-11.b12.el7.x86_64/"}
+      ansible_extra_vars = ansible_extra_vars.merge({ jre_lib_path: "/usr/lib/jvm/java-1.8.0-openjdk-1.8.0.131-11.b12.el7.x86_64/jre/lib", java_home_path: "/usr/lib/jvm/java-1.8.0-openjdk-1.8.0.131-11.b12.el7.x86_64/"})
```

Comment:
@rioh I know this is nitpicky, but would you please put each variable on a separate line like they were before (i.e. newline after the first `{` and newline before the last `}`.  This makes it much easier to see and review, without having to deal with line wrapping.  Same goes for the lines below, thanks!

Reply:
yes sorry, hasty commit